bigquery/snippets/table/bigquery_create_table_range_partitioned.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
```

Comment:
Is this copyright year ok?

Reply:
copy pasting for fun and profit gets me again :)